src/Memento.jl

```diff
@@ -60,9 +61,22 @@ const _loggers = Dict{AbstractString, Logger}(
 
 const LOGGER = getlogger(@__MODULE__)
 
+# Cached values either to lazily load or precompile.
+const _default_formatter = DefaultFormatter(DEFAULT_FMT_STRING)
+const _localtz = Ref{Union{Dates.TimeZone, Nothing}}(nothing)
+
+function getlocalzone()
+    if _localtz[] === nothing
+        global _localtz[] = localzone()
+    end
+
+    return _localtz[]
+end
```

Comment:
Can we do:
```suggestion
function getlocalzone()
    @isdefined(_localtz) || (global _localtz[] = localzone())
    return _localtz
end
```

Reply:
I might just drop this caching behaviour is it's kind of tangental to the primary issue and it's causing tests to fail on linux.

Reply:
Removed and doesn't impact load performance.